indico/web/forms/fields.py

```diff
@@ -308,22 +308,24 @@ def _convert_data(self, data):
         return map(self._get_event_person, data)
 
     def _create_event_person(self, data):
-        user = User.find_first(~User.is_deleted, User.all_emails.contains(data['email'].lower()))
         title = next((x.value for x in UserTitle if data.get('title') == x.title), None)
-        return EventPerson(event_new=self.event, user=user, email=data['email'], _title=title,
-                           first_name=data['firstName'], last_name=data['familyName'],
+        return EventPerson(event_new=self.event, email=data['email'], _title=title,
+                           first_name=data.get('firstName'), last_name=data['familyName'],
```

Comment:
Why is the `familyName` accessed within the `dict` and not using the `get` method? It is guaranteed that it will always be present?